synternet/chain.json

```diff
@@ -0,0 +1,152 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "synternet",
+  "chain_id": "synternet-1",
+  "website": "https://www.synternet.com/",
+  "pretty_name": "Synternet",
+  "status": "live",
+  "network_type": "mainnet",
+  "bech32_prefix": "synt",
```

Comment:
Please register this prefix: https://github.com/satoshilabs/slips/blob/master/slip-0173.md